lib/usd/translators/meshReader.cpp

```diff
@@ -79,10 +79,11 @@ bool MayaUsdPrimReaderMesh::Read(UsdMayaPrimReaderContext& context)
 
     auto    parentNode = context.GetMayaNode(prim.GetPath().GetParentPath(), true);
     MObject transformObj;
-    // Note: in prototype instances, we don't need an intermediary transform node.
+    // Note: in prototype instances, we don't need an intermediary transform node,
+    //       unless the prim has transforms that need to be preserved.
     //       See: EMSUSD-80
     const bool inPrototype = UsdPrim::IsPathInPrototype(prim.GetPath());
-    if (inPrototype) {
+    if (inPrototype && !UsdMayaTranslatorUtil::HasXformOps(prim)) {
         transformObj = parentNode;
     } else {
         bool retStatus = UsdMayaTranslatorUtil::CreateTransformNode(
```

Comment:
This is the key fix: when the prototype has transform on the prim, a Maya transform node must be created to hold that transform.